### DISCUSSION TITLE
Unable to log in to my Vercel account.

### DISCUSSION THREAD

**Discussion by Pathsis** [vercel/community#6357] [Account Issue] [answered]
### Summary

Vercel account tied to Github cannot be logged in.

### Steps to Reproduce

I registered my Vercel account with my Github account. But today, when I logged in to Vercel with my Github account, it didn't pass the authentication and prompted me "There is no Vercel account linked with this GitHub account." I think the problem might be that I added an email address to my Github account two days ago (but the primary email address didn't change). 

I deleted the added mailbox, but I still can't log in to Vercel. what should I do?

**Comment by amyegan** [comment]
Hey @Pathsis. Have you tried logging in with your primary email address instead? Please let me know whether or not that works for you.

**Reply by Pathsis** [reply, marked as answer]
Thank you for your reply! I had tried to log in on my pc using my email, but nothing happened. Then I tried to open Vercel on my phone and found that I was able to log in properly, so I rebound my Github Authentication. Problem solved!

One question is, how do I set a password for the email login? I was unable to find this option.

Thanks again.

**Reply by amyegan** [reply]
We use passwordless login so you will not need to set a password for login. 🙂
